node-api/handlers/beacon/withdrawal.go

```diff
@@ -39,18 +41,22 @@ func (h *Handler) GetPendingPartialWithdrawals(c handlers.Context) (any, error)
 		return nil, err
 	}
 
-	partialWithdrawals, err := h.backend.PendingPartialWithdrawalsAtSlot(slot)
+	st, _, err := h.backend.StateAtSlot(slot)
 	if err != nil {
 		return nil, err
 	}
 
-	st, _, err := h.backend.StateAtSlot(slot)
+	// Get the fork version.
+	forkVersion, err := st.GetFork()
 	if err != nil {
 		return nil, err
 	}
 
-	// Get the fork version.
-	forkVersion, err := st.GetFork()
+	if version.IsBefore(forkVersion.CurrentVersion, version.Electra()) {
+		return nil, types.ErrInvalidRequest
```

Comment:
nit: should we extend the error with something like: `fmt.Errorf("%w: Electra fork not active yet", types.ErrInvalidRequest)`